packaging/package.sh

```diff
@@ -24,14 +24,14 @@ binDir="${BASEDIR}/root/Applications/CodeReady Containers.app/Contents/Resources
 version=$(cat "${BASEDIR}/VERSION")
 
 sign "${binDir}/crc"
-sign "${binDir}/crc-admin-helper-darwin"
+sign "${binDir}/admin-helper-darwin"
 sign "${binDir}/crc-driver-hyperkit"
 
 sign "${BASEDIR}/root/Applications/CodeReady Containers.app"
 
 codesign --verify --verbose "${binDir}/hyperkit"
 
-sudo chmod +sx "${binDir}/hyperkit" "${binDir}/crc-admin-helper-darwin" "${binDir}/crc-driver-hyperkit"
```

Comment:
why these changes are part of this PR?

Reply:
I must have messed up the rebase, thanks for catching it!